xwiki-platform-core/xwiki-platform-localization/xwiki-platform-localization-rest/xwiki-platform-localization-rest-api/src/main/resources/xwiki.rest.localization.model.xsd

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ *
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ *
+-->
+<schema xmlns="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.xwiki.org/localization"
+  xmlns:localization="http://www.xwiki.org/localization" elementFormDefault="qualified">
+  
+  <!--
+  *
+  * Types
+  *
+  -->
+  
+  <complexType name="Translation">
+    <annotation>
+      <documentation>Stores the information of a translation. Currently this includes the
+        translation key, and the raw value of the translation in a requested locale.
+      </documentation>
+    </annotation>
+    <sequence>
+      <!-- The translation key. -->
+      <element name="key" type="string"/>
+      <!-- The raw value of the translation. In other words, the translation value where the parameters are not 
+      resolved. -->
+      <element name="raw" type="string" minOccurs="0"/>
+    </sequence>
+  </complexType>
+
+  <complexType name="Translations">
+    <annotation>
+      <documentation>Stores a collection of resolved translations.</documentation>
+    </annotation>
+    <sequence>
+      <!-- A collection of resolved translations. -->
+      <element name="translations" type="localization:Translation" minOccurs="0" maxOccurs="unbounded"/>
+    </sequence>
+  </complexType>
+
+  <!--
+  *
+  * Elements
+  *
+  -->
+
+  <element name="translations" type="localization:Translations"/>
+</schema>
```

Comment:
Could also be moved to a dedicated `-model` module. WDYT?

Reply:
xwiki-platform-localization-rest-api is already a pretty dedicated module IMO.